modules/social_features/social_featured_content/templates/node--featured.html.twig

```diff
@@ -135,6 +137,7 @@
             {% if topic_type %}
               {% embed "node--teaser__field.html.twig" %}
                 {%- block field_icon -%} label {%- endblock -%}
+                {% block svg_title %} <title>{% trans %} Topic type: {% endtrans %}</title> {% endblock %}
```

Comment:
This SVG should be hidden because there's an sr-only line with the same information right after it.

Reply:
All SVG icons displayed via `node--teaser__field.html.twig` template have a `aria-hidden='true'` attribute